models/backbone/vgg.py

```diff
@@ -0,0 +1,191 @@
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+
+from ..utils import conv3x3_group, norm_layer, \
+     kaiming_init, constant_init, load_checkpoint
+
+
+class BasicBlock(nn.Module):
+    def __init__(self,
+                 inplanes,
+                 planes,
+                 stride=1,
+                 dilation=1,
+                 use_gn=False):
+        super(BasicBlock, self).__init__()
+
+        self.conv = conv3x3_group(
+            inplanes, planes, stride, dilation)
+
+        norm_layers = []
+        norm_layers.append(norm_layer(planes, use_gn))
+        self.norm_names = 'bn' if not use_gn else 'gn'
+        for name, layer in zip(self.norm_names, norm_layers):
+            self.add_module(name, layer)
+
+        self.relu = nn.ReLU(inplace=True)
+        self.maxpool = nn.MaxPool2d(kernel_size=2, stride=2)
```

Comment:
maybe add a parameter `maxpool=True` in this block is better, cause in VGG, not all the conv layer are followed by a `maxpool` layer. Actually, I suggest to delete this `BasicBlock`, because it just a `conv+norm+relu` pairs, this function has been implemented in '..util/layers/ConvModule', you can use `ConvModule` directly. 
Add a `_make_vgg_layer` function is enough.